_learning_hours/ide/shortcuts_vs_resharper_rider.md

```diff
@@ -0,0 +1,61 @@
+---
+theme: shortcuts
```

Comment:
The actual content of the LH is excellent and well explained. I would not put it in a new folder or theme though. I think 'Working in an Ensemble' is a reasonable place to put it. We could also rename 'Git' to 'Tools'.